capitaliq/src/test/scala/com/quantemplate/capitaliq/commands/IdentifierLoader.scala

```diff
@@ -0,0 +1,134 @@
+package com.quantemplate.capitaliq.commands
+
+import munit.FunSuite
+import org.mockito.Mockito.*
+import org.mockito.ArgumentMatchers.*
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
+import akka.actor.typed.ActorSystem
+import java.nio.file.Paths
+import cats.syntax.option.*
+
+import com.quantemplate.capitaliq.qt.QTService
+import com.quantemplate.capitaliq.domain.CapitalIQ.Identifier
+import com.quantemplate.ActorSystemSuite
+import IdentifierLoader.*
+
+class IdentifierLoaderSpec extends ActorSystemSuite:
+  test("given a column name it should properly load the identifiers from QT dataset") {
+    given sys: ActorSystem[Nothing] = actorSystem()
+    given ExecutionContext = sys.executionContext
```

Comment:
I don't think that naming variable as class name is a good convention. please start with lower case -  `executionContext`

Reply:
`ExecutionContext` is a type name, it's only used in implicit context so there is no variable name

Reply:
ah nice scala 3 trick ;-)